README.md

```diff
@@ -1,34 +1,48 @@
 # deepcell-applications
 
-A template of how to create and combine deep learning workflows using `deepcell-tf` and Docker.
+[![Build Status](https://github.com/vanvalenlab/deepcell-applications/workflows/build/badge.svg)](https://github.com/vanvalenlab/deepcell-applications/actions)
+[![Coverage Status](https://coveralls.io/repos/github/vanvalenlab/deepcell-applications/badge.svg?branch=master)](https://coveralls.io/github/vanvalenlab/deepcell-applications?branch=master)
+[![Apache 2.0](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](https://github.com/vanvalenlab/deepcell-applications/blob/master/LICENSE)
+
+A script and runnable Docker image for plugging DeepCell Applications (like `Mesmer`) into existing pipelines.
 
 ## Running the Python script
 
-An example Python script `main.py` is provided as an example `deepcell.applications` workflow.
+The `run_app.py` script is used to read the input files from the user and process them with the selected Application.
+An example Python script `run_app.py` is provided as an example `deepcell.applications` workflow.
 
 ### Script arguments
 
-This script uses `argparse` to process command line arguments. Below is a table summarizing the arguments and any defaults. For more information, use `python main.py --help`.
+The first required argument to the script is the Application name: `python run_app.py APP_NAME`.
+Each supported application has a variety of different configuration arguments.
+Below is a table summarizing the currently supported applications and their arguments and any defaults.
+For more information, use `python run_app.py --help` or `python run_app.py APP_NAME --help`.
+
+#### Mesmer arguments
 
 | Name | Description | Default Value |
 | :--- | :--- | :--- |
-| `--nuclear-image` | **REQUIRED**: The path to a nuclear channel image. | `""` |
-| `--membrane-image` | The path to a membrane channel image. | `""` |
 | `--output-directory` | Directory to save output file. | `"./output"` |
 | `--output-name` | The name for the output file. | `"mask.tif"` |
+| `--nuclear-image` | **REQUIRED**: The path to a nuclear channel image. | `""` |
+| `--nuclear-channel` | The channel of `nuclear-image` to select. If multiple values are passed, the channels will be summed. | `0` |
+| `--membrane-image` | The path to a membrane channel image. If not passed, an array of zeroes will be used instead. | `""` |
+| `--membrane-channel` | The channel of `membrane-image` to select. If multiple values are passed, the channels will be summed. | `0` |
 | `--compartment` | Predict nuclear or whole-cell segmentation. | `"whole-cell"` |
+| `--image-mpp` | The resolution of the image in microns-per-pixel. | `0.5` |
```

Comment:
Standardizing sounds good. What do you have in mind?

I had envisioned that the kiosk-frontend will just map 20x to 0.5 mpp and 40x to 1, but still use image_mpp values on the backend

Reply:
For sure, I think the standardization would be all of the different ways of using it have the same rescale option. So if we change the frontend to be magnification based, we can use the same settings here as well.

Maybe we could even add a helper function to the underlying Mesmer app that takes in 10X, 20X, etc, and computes what the resize value should be. But that is optional. Main thing is that users see the same set of options no matter which entrypoint to the model they're using.

Reply:
OK. I'm open to redesigning the application resizing interface, but I think that is another scope of work.

To be consistent, I think we should continue to use mpp, even if it's a bit harder to input. I've updated the help menu to equate the default (0.5) to ~20x. I'm hopeful that this will be helpful enough.

Reply:
Agreed. If/when we update deepcell-tf application to handle magnification-based specification, can update args here as well